iaso/api/org_unit_change_requests/views.py

```diff
@@ -88,6 +88,12 @@ def get_queryset(self):
             )
             .exclude_soft_deleted_new_reference_instances()
         )
+
+        if not self.request.query_params.get("source_version_id"):
+            org_units_change_requests = org_units_change_requests.filter(
+                org_unit__version=self.request.user.iaso_profile.account.default_version
+            )
```

Comment:
You should add `org_unit__version` in the `select_related`.

It seems that `source_version_id` is now mandatory. So shouldn't we instead [make the `source_version_id` mandatory](https://django-filter.readthedocs.io/en/stable/ref/filters.html#required)?

This would save us this query since the front-end already knows the `default_version`.